### PR TITLE
Support user id in group and room

### PR DESCRIPTION
**Pull request by nugsazz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -101,4 +101,4 @@ ENV/
 .idea
```
Comment (nugsazz) [inline .gitignore]: q.jangkung

File: README.rst [hunk 2]
```diff
@@ -534,9 +534,11 @@ Source
 - SourceGroup
     - type
     - group\_id
+    - user\_id
```
Comment (nugsazz) [inline README.rst]: q.jangkung

File: .gitignore (hunk 1 above)
Comment (tokuhirom) [inline .gitignore]: what's "q.jangkung"?

Comment (nugsazz) [inline .gitignore]: http://line.me/R/ti/g/IGk-gIWVl9

**Comment by be-hase** [comment]
@nugsazz 
these commits are already merged, so I close this PR.

**Pull request closed without merging**